### ISSUE TITLE
Errors that occur in my own data

### ISSUE THREAD

**Issue by enomto** [divamgupta/image-segmentation-keras#27]
File "train.py", line 79, in <module>
    m.fit_generator( G , 871  , validation_data=G2 , validation_steps=1 ,  epochs=1 )
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/legacy/interfaces.py", line 91, in wrapper
    return func(*args, **kwargs)
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/engine/training.py", line 2262, in fit_generator
    workers=0)
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/legacy/interfaces.py", line 91, in wrapper
    return func(*args, **kwargs)
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/engine/training.py", line 2377, in evaluate_generator
    generator_output = next(output_generator)
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/utils/data_utils.py", line 779, in get
    six.reraise(value.__class__, value, value.__traceback__)
  File "/home/atsushi/.local/lib/python2.7/site-packages/keras/utils/data_utils.py", line 644, in _data_generator_task
    generator_output = next(self._generator)
  File "/home/atsushi/keras/work/image-segmentation-keras/LoadBatches.py", line 69, in imageSegmentationGenerator
    assert len( images ) == len(segmentations)
AssertionError

Is there a problem with Labels?

**Comment by duyanfang123** [comment]
Hi ,how to train my own datasets ?
